hal/targets.json

```diff
@@ -1015,14 +1013,14 @@
         "macros_add": ["TARGET_MCU_NRF51_16K_S110"],
         "EXPECTED_SOFTDEVICES_WITH_OFFSETS": [
             {
-                "name"   : "s110_nrf51822_8.0.0_softdevice.hex",
-                "boot"   : "s110_nrf51822_8.0.0_bootloader.hex",
-                "offset" : 98304
+                "name": "s110_nrf51822_8.0.0_softdevice.hex",
+                "boot": "s110_nrf51822_8.0.0_bootloader.hex",
+                "offset": 98304
             },
             {
-                "name"   : "s110_nrf51822_7.1.0_softdevice.hex",
-                "boot"   : "s110_nrf51822_7.1.0_bootloader.hex",
-                "offset" : 90112
+                "name": "s110_nrf51822_7.1.0_softdevice.hex",
+                "boot": "s110_nrf51822_7.1.0_bootloader.hex",
+                "offset": 90112
```

Comment:
Sadly, JSON doesn't support hex constants :( We can write it as `0x90112`, but then we need to know when we parse the JSON that the string is actually a hex constant that needs to be parsed. Not sure if it's worth the trouble.